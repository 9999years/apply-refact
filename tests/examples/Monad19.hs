yes = mapM async ds >>= mapM wait >> return ()