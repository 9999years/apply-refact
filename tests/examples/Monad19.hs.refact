[("tests/examples/Monad19.hs:1:7-33: Warning: Use mapM_\nFound:\n  mapM async ds >>= mapM wait\nPerhaps:\n  mapM async ds >>= mapM_ wait\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 25, endLine = 1, endCol = 29}, subts = [], orig = "mapM_"}])]
