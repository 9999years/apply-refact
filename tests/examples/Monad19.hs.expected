yes = mapM async ds >>= mapM_ wait >> return ()
